aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -0,0 +1,132 @@
+              setActiveHeadingId(entry.target.id);
+            }
+          });
+        }, options);
+        elements.forEach(el => {
+          if (el) {
+            observer.current?.observe(el);
+          }
+        });
+      }
+      return () => observer.current?.disconnect();
+    };
+  }, [headings, options]);
+  return activeHeadingId;
+};
+
+export const InPageNavigation = ({ title }) => {
+  const match = siteContents.find(
+    item => item.name.toLowerCase() === title.toLowerCase(),
+  );
+
+  const theme = useContext(ThemeContext);
+  const regexp = new RegExp(/#{1,} (...+?) ?~{2}/, 'g');
+  const headings = match && [...match.content.matchAll(regexp)];
+
+  // top and bottom margin values to calculate intersection window
+  const topMargin = '50';
+  const bottomMargin = '-95';
+  const activeId = useActiveHeadingId(headings, {
+    rootMargin: `${topMargin}% 0% ${bottomMargin}% 0%`,
+  });
+
+  return headings.length > 0 ? (
+    <Box
+      pad={{ horizontal: 'xsmall' }} // pad for keyboard focus
+      style={{
+        height: 'calc(100vh - 50px)',
+        overflowY: 'auto',
+        position: 'sticky',
+        marginTop: '123px', // align "Jump to section" with page title at start
+        flexShrink: 0,
+        top: '30px',
+      }}
+      width="small"
+      flex={false}
+    >
+      <Box
+        pad="small"
+        flex={false}
+        border={{ side: 'left', color: 'border-weak', size: 'small' }}
+      >
+        <Text color="text-strong" weight="bold">
+          Jump to section
+        </Text>
+      </Box>
+
+      <Nav gap="none" a11yTitle="Jump to section">
+        {headings.map((heading, index) => {
+          const levelRegexp = new RegExp(/^(#)+/);
+          const [level] = heading[0].match(levelRegexp);
+          // heading[1] refers to the full heading title matched by regex
+          const headingTitle = heading[1];
+          const active = activeId === nameToSlug(headingTitle);
+
+          const borderLeft = {
+            side: 'left',
+            size: active ? 'medium' : 'small',
+            color: active ? 'border-strong' : 'border-weak',
+          };
+
+          let subsectionPad = 'xsmall';
+          if (level.length > 3) subsectionPad = 'large';
+          else if (level.length === 3) subsectionPad = 'medium';
+          return (
+            <Box border={borderLeft}>
+              <Link key={index} href={`#${nameToSlug(headingTitle)}`} passHref>
+                <Button
+                  style={{ textAlign: 'start' }}
+                  size="small"
+                  label={
+                    <Box
+                      pad={{ left: subsectionPad }}
+                      margin={
+                        active
+                          ? undefined
+                          : { left: theme.global.borderSize.small }
+                      }
+                    >
+                      <Text color="text-strong" size="small" weight="normal">
```

Comment:
Do we need `weight="normal"` or is that the default?

Reply:
`weight = "normal"` is needed, since otherwise the text becomes bolded similar to that of the "Jump to section" title